media-library/overview/rclone.md

```diff
@@ -0,0 +1,138 @@
+# Rclone
+
+![](../../.gitbook/assets/rclone/rclone-logo.svg)
+
+[Rclone](https://rclone.org/) is a command-line program that helps manage files on a cloud storage. It supports [over 70 cloud storage products](https://rclone.org/#providers) and has powerful cloud equivalents to the unix commands rsync, cp, mv, mount, ls, ncdu, tree, rm, and cat.
+
+ImageKit natively integrates with rclone to offer a seamless experience in managing your media library assets via the command-line and opens up doors to many advanced workflows.
+
+In this guide, we would be looking at how you can configure ImageKit's Media Library as a cloud storage provider to rclone.
+
+## Getting Started
+
+Before you start, make sure you have an ImageKit account.
+[Sign up](https://imagekit.io/registration/) for a free plan, starting with generous usage limits and when your requirements grow, you can easily upgrade to a plan that best fits your needs. More pricing information is available [here](https://imagekit.io/plans).
+
+Also, this guide assumes that you have rclone set up in your system. You can check out the [Rclone install guide](https://rclone.org/install/) to install it based on your operation system i.e. Linux / macOS / Windows.
```

Comment:
Consistently use `Rclone` instead of `rclone`. This feedback is for whole page.

Reply:
Done